dotnet/src/IntegrationTests/Connectors/OpenAI/OpenAICompletionTests.cs

```diff
@@ -409,6 +411,58 @@ public async Task MultipleServiceLoadPromptConfigTestAsync()
         Assert.Contains("Pike Place", azureResult.GetValue<string>(), StringComparison.OrdinalIgnoreCase);
     }
 
+    [Fact]
+    public async Task UsingStandardizedChatPromptGivesNoXmlBackAsync()
+    {
+        // Arrange
+        var builder = this._kernelBuilder.WithLoggerFactory(this._logger);
+        this.ConfigureAzureOpenAIChatAsText(builder);
+        Kernel target = builder.Build();
+
+        // Create prompt
+        KernelFunction prompt = KernelFunctionFactory.CreateFromPrompt(@"
+            <message role=""system"">You are a helpful assistant.</message>
+            <message role=""user"">How many 20 cents can I get from 1 dolar?</message>
+        ");
+
+        var fullContent = new StringBuilder();
+        await foreach (var content in target.InvokeStreamingAsync<StreamingChatMessageContent>(prompt))
+        {
+            fullContent.Append(content);
+        }
+        fullContent.Append((await target.InvokeAsync(prompt)).GetValue<string>());
+
+        Assert.False(fullContent.ToString().Contains('<', StringComparison.Ordinal));
+        Assert.False(fullContent.ToString().Contains('>', StringComparison.Ordinal));
+    }
+
+    [Fact]
+    public async Task UsingStandardizedChatPromptAgainstTextCompletionGivesXmlBackAsync()
```

Comment:
I would recommend being careful with integration testing here. During my manual testing, sometimes AI returned response in XML and sometimes as plain text, so I wouldn't rely on AI response in this case, otherwise this test can randomly fail.

Also, we can test this scenario with unit testing. The test case can verify if specific method in Connector receives parsed chat history or plain text as input parameter.